utils/utils.go

```diff
@@ -68,19 +66,21 @@ func StringSliceDiff(a, b []string) []string {
 	return result
 }
 
-func GetIpAddress(r *http.Request) string {
+func GetIpAddress(r *http.Request, trustedProxyIPHeader []string) string {
 	address := ""
 
-	header := r.Header.Get(model.HEADER_FORWARDED)
-	if len(header) > 0 {
-		addresses := strings.Fields(header)
-		if len(addresses) > 0 {
-			address = strings.TrimRight(addresses[0], ",")
+	for _, proxyHeader := range trustedProxyIPHeader {
```

Comment:
Looks like `HEADER_REAL_IP` and `HEADER_FORWARDED` have different encoding semantics, based on the old parsing behaviour. I /think/ the `Fields` and `TrimRight` amount to the same thing, but I wonder if we want to be explicit and handle each type instead? And do we really want to try to parse other headers which potentially different encoding semantics?

Reply:
I've considered it as well, but the problem is that `X-Real-Ip` is not standardized and I couldn't find a lot of information about it. I intentionally decided against implementing the special case, since the current code works fine with `X-Real-Ip` as well:
https://play.golang.org/p/eF0H4uLEsE1

Since it's not standardized some people might use it in a similar fashion, where this approach would still work, while the previous one would break.